### PR TITLE
Transfer+Misc FeatureFlags

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15905

*Description of changes:*
* Implements addition FF for Transfer Ownership changes
* Implements future feature flag placeholders
* Minor unit test update.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1293-i106vu27.web.app

File: ppr-ui/src/utils/feature-flags.ts [hunk 1]
```diff
@@ -9,8 +9,11 @@ export const defaultFlagSet: LDFlagSet = {
   'bcregistry-ui-mhr-enabled': false,
   'search-registration-number': false,
   'search-serial-number': false,
-  'mhr-ui-enabled': false,
-  'mhr-registration-enabled': false,
+  'mhr-ui-enabled': false, // Enables MHR search options
+  'mhr-registration-enabled': false, // Enables MHR table tab
+  'mhr-transfer-enabled': false, // Enables changes to base MHR Home Owners within the MHR Information flow
```
Comment (dimak1) [inline ppr-ui/src/utils/feature-flags.ts:14]: So to properly enable transfers we need to enable **mhr ui** first + **mhr transfers**, right?

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/utils/feature-flags.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/feature-flags.ts:14]: Mostly true yes! Tho `mhr-ui-enabled` is for search features.

In order to get to the Transfer, `mhr-registration-enabled` would need to be `true` for the user to access the Registration via the Table etc

**Comment by cameron-eyds** [comment]
Will hold off on a merge until tomorrow when Melissa is back, as this may require additional LD configuration. 
Don't want to block dev if we can't update the key values.

**Pull request merged**